src/engine/strat_engine/filesystem.rs

```diff
@@ -3,17 +3,16 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 extern crate rand;
 
-use devicemapper::DM;
-
 use consts::IEC;
```

Comment:
I think consts belongs to Stratisd, so should go below devicemapper.

Which reminds me, shouldn't you remove ```SECTOR_SIZE``` from consts.rs as well in this PR?

I bet that eventually, consts::IEC should end up in devicemapper as well. It might be worth an issue, "move IEC to devicemapper", but no need to put it in this PR.